packages/browser-cli/src/session.test.ts

```
@@ -2,6 +2,29 @@ import { describe, expect, it, vi } from "vitest";
 import { CdpClient } from "./cdp";
 import { CdpBrowserCliSession } from "./session";
 
+const CONTROL_KEY = "__MSW_DEV_TOOL_CONTROL__";
+
+const createEvaluatingClient = (bridge: Record<string, unknown>) => {
+  const call = vi.fn(async (_method: string, parameters: { expression: string }) => {
+    const root = globalThis as Record<string, unknown>;
+    const previousBridge = root[CONTROL_KEY];
+    root[CONTROL_KEY] = bridge;
+    try {
+      return { result: { value: (0, eval)(parameters.expression) } };
```

Biome (2.5.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)